handbook/growth.md

```diff
@@ -43,15 +43,15 @@ Once a blog post has been written, approved, and published, ensure that it is pr
 
 ## Press releases
 
-If we are doing a press release, we are probably pitching it to one or more reporters as an exclusive story if they choose to take it.  Consider not sharing or publicizing any information related to the upcoming press release before the announcement.  See also https://www.quora.com/What-is-a-press-exclusive-and-how-does-it-work
+If we are doing a press release, we are probably pitching it to one or more reporters as an exclusive story if they choose to take it.  Consider not sharing or publicizing any information related to the upcoming press release before the announcement.  Also see: https://www.quora.com/What-is-a-press-exclusive-and-how-does-it-work
```

Comment:
@timmy-k, can you make this link a related word or phrase for accessibility features to find?
"Also see: https://www.quora.com/What-is-a-press-exclusive-and-how-does-it-work"